rewrite-java/src/main/java/org/openrewrite/java/UseStaticImport.java

```diff
@@ -58,7 +59,7 @@ private class UseStaticImportVisitor extends JavaIsoVisitor<ExecutionContext> {
         public J.MethodInvocation visitMethodInvocation(J.MethodInvocation method, ExecutionContext ctx) {
             MethodMatcher methodMatcher = new MethodMatcher(methodPattern);
             J.MethodInvocation m = super.visitMethodInvocation(method, ctx);
-            if (methodMatcher.matches(m)) {
+            if (methodMatcher.matches(m) && method.getMethodType().hasFlags(Flag.Static)) {
                 if (m.getMethodType() != null) {
```

Comment:
```suggestion
            if (methodMatcher.matches(m)) {
                if (m.getMethodType() != null && m.getMethodType().hasFlags(Flag.Static)) {
```

Reply:
Apply suggestions from https://github.com/openrewrite/rewrite/pull/3662/files#r1382445827

Reply:
Turns out it's a little more complicated when we look at 
https://github.com/openrewrite/rewrite/blob/e5c77d5afe17070474d7b5cf6e728600c5f776af/rewrite-java-test/src/test/java/org/openrewrite/java/UseStaticImportTest.java#L76-L78